### DISCUSSION TITLE
Reducing the number of libraries for UFS WM

### DISCUSSION THREAD

**Discussion by arunchawla-NOAA** [ufs-community/ufs-weather-model#1345] [Announcements]
Libraries are critical for UFS WM, but we should limit these to the libraries we really need. The aim of this discussion is to take a close look at the libraries we are loading for the UFS WM and determine if these are really needed. Some may require longer discussions but others may be a lot more straight forward to take care off

**Comment by climbfuji** [comment]
netcdf-cxx is currently part of spack-stack (for JEDI). This can be removed very soon, needs a bit more development on our end (we want to get rid of this, not as well supported and up to date as netcdf-c).
